### PR TITLE
airbyte-to-flow: switch back to stdio

### PR DESCRIPTION
**Pull request by mdibaiee**

### CODE REVIEW

File: airbyte-to-flow/src/connector_runner.rs [hunk 1]
```diff
@@ -150,14 +125,14 @@ pub async fn run_airbyte_source_connector(
         }
 
         exit_status_result
-    });
+    };
```
Comment (williamhbaker) [inline airbyte-to-flow/src/connector_runner.rs:128]: There were some changes made in https://github.com/estuary/airbyte/pull/54 to the code block around `if exit_status_result.is_ok() {` (above) specifically for connectors using TCP. Do we not need to undo those changes as well with the switch back to stdio?

**Comment by williamhbaker** [comment]
This generally makes sense to me on a skim. I was going to try to do a sanity check by comparing the diff of when TCP mode was added. I thought I could find that in https://github.com/estuary/airbyte/pull/34, but there's a lot going on in that PR (I think bringing in the entirety of the airbyte-to-flow code?) - is there a specific commit etc. that would be good for a comparison of what this is specifically undoing? Or just a general description to help understand if the things going on here are broad changes, or just reverting back to something close to a known working state.

**Comment by mdibaiee** [comment]
@williamhbaker the changes here are basically moving towards the logic that was / is in connector-proxy for using `stdio`, see this file for example: https://github.com/estuary/flow/blob/master/crates/connector_proxy/src/connector_runner.rs#L190-L200

airbyte-to-flow is a smaller version of connector-proxy focused solely on airbyte and nothing else. The transition to TCP and airbyte-to-flow happened as part of the same PR as you noticed, so it's hard to disentangle the change, but comparing with connector-proxy is possible.

File: airbyte-integrations/connectors/source-gitlab/Dockerfile [hunk 2]
```diff
@@ -4,8 +4,8 @@ FROM python:3.9-slim
 RUN apt-get update && apt-get install -y bash && rm -rf /var/lib/apt/lists/*
 
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
-#COPY --from=ghcr.io/estuary/airbyte-to-flow:dev /airbyte-to-flow ./
-#
+COPY --from=ghcr.io/estuary/airbyte-to-flow:dev /airbyte-to-flow ./
```
Comment (willdonnelly) [inline airbyte-integrations/connectors/source-gitlab/Dockerfile:7]: nit: Maybe move this down to line 16 so the pattern is consistent across all these Dockerfiles?

**Review by willdonnelly — APPROVED** [review]
LGTM

File: airbyte-to-flow/src/connector_runner.rs (hunk 1 above)
Comment (williamhbaker) [reply airbyte-to-flow/src/connector_runner.rs:128]: I think it makes sense to keep the new code as-is - just wasn't sure if it was equivalent or not. I agree that it seems like a nice improvement.

**Review by williamhbaker — APPROVED** [review]

**Comment by mdibaiee** [comment]
I'll deploy this on Saturday, let there be peace on this Friday evening 😁

**Pull request merged**